src/game.cpp

```diff
@@ -268,6 +269,9 @@ game::game() :
     tileset_zoom(16),
     weather_override( WEATHER_NULL )
 {
+    remoteveh_cache_turn = INT_MIN;
+    temperature = 0;
```

Comment:
Putting them in the initialization list requires you to put them in the correct order, and didn't feel like it.  If there's a reason to move it, I'd be happy to.